doc/api/fs.markdown

```diff
@@ -101,7 +101,7 @@ Objects returned from `fs.watch()` are of this type.
 ### Event: 'change'
 
 * `event` {String} The type of fs change
-* `filename` {String} The filename that changed (if relevant/available)
+* `filename` {String | Buffer} The filename that changed (if relevant/available)
```

Comment:
It will only be a buffer if fs.watch is called with the `encoding` option
set to `'buffer'`
On Mar 17, 2016 8:53 PM, "Ron Korving" <notifications@github.com> wrote:

> In doc/api/fs.markdown
> <https://github.com/nodejs/node/pull/5616#discussion_r56610752>:
>
> > @@ -101,7 +101,7 @@ Objects returned from `fs.watch()` are of this type.
> >  ### Event: 'change'
> >
> >  * `event` {String} The type of fs change
> > -* `filename` {String} The filename that changed (if relevant/available)
> > +* `filename` {String | Buffer} The filename that changed (if relevant/available)
>
> In the case of these events, how can I as a user predict whether it will
> be a string or Buffer?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/5616/files/9d5fe80af91605b9cb126abb0d8238749933f555#r56610752>
>